src/js/rx/actions/prescriptions.js

```diff
@@ -3,12 +3,17 @@ export function loadData(id) {
     // TODO: Use id param instead of test id
     // when API is able to retrieve any individual Rx.
     const testId = 1435525;
-    return dispatch => fetch(`/rx-api/prescriptions/${testId}`)
-      .then(res => res.json())
-      .then(
-        data => dispatch({ type: 'LOAD_PRESCRIPTION_SUCCESS', data }),
+    const rxUrl = `/rx-api/prescriptions/${testId}`;
+    const rxUrls = [rxUrl, `${rxUrl}/trackings`];
+    return dispatch => {
+      Promise.all(rxUrls.map(url => fetch(url).then(res => res.json()))).then(
```

Comment:
This is awesome, but a little hard to follow.  Maybe break it into separate lines?